src/text-input/index.ts

```diff
@@ -231,34 +273,59 @@ export class TextInputBase<P extends TextInputProperties = TextInputProperties>
 		});
 	}
 
+	protected renderHelperText(): DNode {
+		const { properties: { helperText }, _state: { valid, message } } = this;
+		const text = message || helperText;
+
+		return text ? v('div', {
```

Comment:
should this perhaps be an `aside` and the text itself a `span` ? could be wrong, but just thinking out loud.

Reply:
Either is fine with me. I was just using the same markup as in the example [here](https://material.io/develop/web/components/input-controls/text-field/).
<img width="814" alt="screen shot 2019-02-25 at 1 51 46 pm" src="https://user-images.githubusercontent.com/293805/53364488-8412fc80-3904-11e9-9ea0-88cfa70ba247.png">

Reply:
fair enough then!